src/hooks/useGeneratorColor/__stories__/UseGeneratorColor.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import type {Meta, StoryFn} from '@storybook/react';
+
+import {Button} from '../../../components/Button';
+import {Checkbox} from '../../../components/Checkbox';
+import {Select} from '../../../components/Select';
+import {block} from '../../../components/utils/cn';
+import type {UseGeneratorColorProps} from '../types';
+
+import {Color} from './Color';
+import {colorModes} from './constants';
+import {randomString} from './utils/randomString';
+
+import './GeneratorColor.scss';
+
+const b = block('generator-color');
+
+export default {title: 'Hooks/useGeneratorColor'} as Meta;
+
+const DefaultTemplate: StoryFn = () => {
+    const [tokens, setTokens] = React.useState<string[]>([]);
+    const [mode, setMode] = React.useState<string[]>(['unsaturated']);
+    const [withText, setWithText] = React.useState(false);
+
+    const onClick = React.useCallback(() => {
```

Comment:
Let's pregenerate about 10 colors and display them as a table with all mods showing